namespaces/live-1.cloud-platform.service.justice.gov.uk/hello-world-test/00-namespace.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     cloud-platform.justice.gov.uk/environment-name: "development"
   annotations:
     cloud-platform.justice.gov.uk/business-unit: "HMPPS Digital"
-    cloud-platform.justice.gov.uk/slack-channel: "${slack-channel}"
+    cloud-platform.justice.gov.uk/slack-channel: "hello-world-test"
```

Comment:
This needs to be an actual slack channel that exists on the mojdt slack instance. e.g. `#hmpps_dev` would be a good choice.

Reply:
I created a private channel in slack for this purpose, as this is purely for running through the hello world project on the Cloud Platform wiki. Is a private channel unsuitable?